pypeit/scripts/view_fits.py

```diff
@@ -57,42 +57,46 @@ def main(args):
         hdu = fits.open(args.file)
         img = hdu[args.exten].data
         # Write
-        msgs.warn('Writing kludge file to {:s}'.format(kludge_fil))
-        hdunew = fits.PrimaryHDU(img)
-        hdulist = fits.HDUList([hdunew])
-        hdulist.writeto(kludge_fil,clobber=True)
+        ginga.show_image(img)
+#        msgs.warn('Writing kludge file to {:s}'.format(kludge_fil))
+#        hdunew = fits.PrimaryHDU(img)
+#        hdulist = fits.HDUList([hdunew])
+#        hdulist.writeto(kludge_fil,clobber=True)
         #
-        args.file = kludge_fil
+#        args.file = kludge_fil
 
     # RAW_LRIS??
     if args.raw_lris:
         # 
         img, head, _ = keck_lris.read_lris(args.file)
+        ginga.show_image(img)
```

Comment:
Am glad you've eliminated this kludge.
Might as well eliminate the commented lines too.

Reply:
Fixed!